src/index.js

```diff
@@ -22,7 +24,7 @@ const printList = () => {
         tasks += `
           <li id = "${task.index}" class="item">
             <label>
-              <input type="checkbox" name="chk${task.index}" id="chk${task.index}">
+              <input class="checked" type="checkbox" name="chk${task.index}" id="chk${task.index}">
               <input class="edit borderless fit hidden" type="text" name="edit${task.index}" id="edit${task.index}" placeholder="Edit task...">
               <p id="task${task.index}">${task.description}</p>
             </label>
```

Comment:
- [ ] Please know that there is an issue with this function:
  ![image](https://user-images.githubusercontent.com/20567503/196300471-06898f7e-e76b-42b9-854e-cf6201e31f51.png)
  I wasn't able to test anything, so kindly fix this issue so we can test the functionalities of your project.